components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -87,6 +87,7 @@ open class BrowserMenu internal constructor(
 
             displayPopup(view, anchor, orientation).also {
                 currAnchor = anchor
+                currAnchor?.addOnAttachStateChangeListener(this@BrowserMenu)
```

Comment:
Null safety `?.` is unnecessary here. I would use `anchor` instead of `currAnchor` to attach the state change listener, before we assign the currAnchor val.

Reply:
I wanted to use the same reference used in other places of the code.